src/ui/ImageRenderer/index.tsx

```diff
@@ -95,17 +101,17 @@ const ImageRenderer = ({
     );
   }, [url]);
 
-  return (
+  return dynamicMinWidth && dynamicMinHeight && (
     <div
       className={[
         ...(Array.isArray(className) ? className : [className]),
         'sendbird-image-renderer',
       ].join(' ')}
       style={{
         width: '100%',
-        minWidth: parsedWidth,
-        maxWidth: fixedSize ? parsedWidth : '400px',
-        height: parsedHeight,
+        minWidth: dynamicMinWidth,
+        maxWidth: fixedSize ? dynamicMinWidth : '400px',
```

Comment:
IDK the details but this condition looks a bit awkward. 

> Fixed ? then use fixed size, if not, use dynamic width size 

```suggestion
        maxWidth: fixedSize ?  '400px' : dynamicMinWidth,
```

Reply:
yes its does sound awkard and thought the same way. however, its follows the original logic and it seems the 'fixedSize' is meant for 'no zoom' when image size is fixed. Hard to explain in words but its like zooming image inside fixed image window.

Reply:
Ahh so the `fixedSize` is for the image container not the image itself. Modifying the variable name to make it more self-descriptive or leaving a comment would be helpful though.

Reply:
I agree but i have no idea. Any suggestion for the better name?

Reply:
Hmm fixedImageWindowSize? Up to you. It's not a strong opinion and a blocker :)

Reply:
Hmm I prefer that but lets just leave it as is bc `ImageRendererProps` is already publically available so don't want to risk any breaking change.